### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ Before you get started with Removable Storage Access Control, you must confirm y
...
 
-3. Save both rule and group XML files on network share folder and put network share folder path into the Group Policy setting: **Computer Configuration -> Administrative Templates -> Windows Components -> Microsoft Defender Antivirus -> Device Control: ‘Define device control policy groups’ and ‘Define device control policy rules’**.
+3. Save both rule and group XML files on network share folder and put network share folder path into the Group Policy setting: **Computer Configuration -> Administrative Templates -> Windows Components -> Microsoft Defender Antivirus -> Device Control: ‘Define device control policy groups’ and ‘Define device control policy rules’**. If you can not find the policy configuration UX in the Group Policy, you can download the [WindowsDefender.adml](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.adml) and [WindowsDefender.admx](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.admx) file by clicking 'Raw' and 'Save as'.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
3. Save both rule and group XML files on the network share folder and put the network share folder path into the Group Policy setting: **Computer Configuration -> Administrative Templates -> Windows Components -> Microsoft Defender Antivirus -> Device Control: ‘Define device control policy groups’ and ‘Define device control policy rules’**. If you cannot find the policy configuration UX in the Group Policy, you can download the [WindowsDefender.adml](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.adml) and [WindowsDefender.admx](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.admx) files by selecting **Raw** and then **Save as**.
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 2]
```diff
@@ -331,6 +331,11 @@ Another reason could be that the XML file is not correctly formatted, for exampl
 
 If there is a value and the policy is managed via Group Policy, check whether the client device can access the policy XML path.
 
+**There is no configuration UX for ‘Define device control policy groups’ and ‘Define device control policy rules’ on my Group Policy**
+
+We do not backport the Group Policy configuation UX, but you can still get related adml and admx files by clicking 'Raw' and 'Save as' the [WindowsDefender.adml](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.adml) and [WindowsDefender.admx](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.admx) files.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
We do not backport the Group Policy configuration UX, but you can still get related adml and admx files by selecting **Raw** and then **Save as** to download the [WindowsDefender.adml](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.adml) and [WindowsDefender.admx](https://github.com/microsoft/mdatp-devicecontrol/blob/main/Removable%20Storage%20Access%20Control%20Samples/WindowsDefender.admx) files.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, kindly commit my suggested changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
Gentle reminder @tewchen

**Pull request merged**